### PR TITLE
fix: ansible verbosity

### LINKED ISSUE #2627: Ansible: verbosity level 0 doesn't seem to work
## Bug Report

**What did you do?**
I tried using verbosity level 0 via:
- annotation on the CR:
  "ansible.operator-sdk/verbosity": "0"
- operator args:
            args:
            - "--ansible-verbosity"
            - "0"
to control what is getting into the logs, but it doesn't seem to work i.e. getting operator errors like:

`{"level":"error","ts":1583486711.1724753,"logger":"runner","msg":"usage: ansible-runner [-h] [--version] [-m MODULE | -p PLAYBOOK | -r ROLE]\n                      [-b BINARY] [--hosts HOSTS] [-i IDENT]\n                      [--rotate-artifacts ROTATE_ARTIFACTS]\n                      [--roles-path ROLES_PATH] [--role-vars ROLE_VARS]\n                      [--role-skip-facts] [--artifact-dir ARTIFACT_DIR]\n                      [--project-dir PROJECT_DIR] [--inventory INVENTORY]\n                      [--forks FORKS] [-j] [-v] [-q] [--cmdline CMDLINE]\n                      [--debug] [--logfile LOGFILE] [-a MODULE_ARGS]\n                      [--process-isolation]\n                      [--process-isolation-executable PROCESS_ISOLATION_EXECUTABLE]\n                      [--process-isolation-path PROCESS_ISOLATION_PATH]\n                      [--process-isolation-hide-paths PROCESS_ISOLATION_HIDE_PATHS]\n                      [--process-isolation-show-paths PROCESS_ISOLATION_SHOW_PATHS]\n                      [--process-isolation-ro-paths PROCESS_ISOLATION_RO_PATHS]\n                      [--directory-isolation-base-path DIRECTORY_ISOLATION_BASE_PATH]\n                      [--limit LIMIT]\n                      {run,start,stop,is-alive} private_data_dir\nansible-runner: error: argument command: invalid choice: '' (choose from 'run', 'start', 'stop', 'is-alive')\n","job":"2015796113853353331","name":"(I removed)","namespace":"(I removed)","error":"exit status 2","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\tpkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\ngithub.com/operator-framework/operator-sdk/pkg/ansible/runner.(*runner).Run.func1\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/runner/runner.go:217"}
{"level":"error","ts":1583486711.1727486,"logger":"reconciler","msg":"Failed to get ansible-runner stdout","job":"2015796113853353331`

**What did you expect to see?**
My plan was to have the ability to control amount of logging as with verbosity 1 there's quite a lot of data printed from a single run of my playbook. For example when looping over a list containing few dozens of big dictionaries, there's all printed and even if I use no_log: True it still prints this useless line "output was censored" for each item in a loop...
With level 0 logging by default I would still be getting each loop item printed, but could use loop_control to print something useful & harmless (since there seem to be no way to stop printing items in a loop in Ansible...), and so I could limit the output to just item.name so a single dict key would be printed and not the whole dict.

**What did you see instead? Under which circumstances?**
Errors occur and reconciliation is not happening at all.

**Environment**
* operator-sdk version: 0.15.2

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- Fix ansible verbosity annotation

**Motivation for the change:**

Note: If this PR is fixing an issue make sure to add a note saying:
Closes #2627

### CODE REVIEW

File: pkg/ansible/runner/runner_test.go [hunk 1]
```diff
@@ -209,17 +209,17 @@ func TestAnsibleVerbosityString(t *testing.T) {
 		verbosity      int
 		expectedString string
 	}{
-		{verbosity: -1, expectedString: ""},
-		{verbosity: 0, expectedString: ""},
```
Comment (fabianvf) [inline pkg/ansible/runner/runner_test.go]: This looks correct to me, if verbosity is 0 no `-v` is passed

File: pkg/ansible/runner/runner.go [hunk 2]
```diff
@@ -59,9 +59,12 @@ type Runner interface {
 
 func ansibleVerbosityString(verbosity int) string {
 	if verbosity > 0 {
-		return fmt.Sprintf("-%v", strings.Repeat("v", verbosity))
+		// the default verbosity 0 is == -v
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply pkg/ansible/runner/runner.go]: Hun .. I think I misunderstood it. Thank you for your check. I was looking for a second look as well :-) I will check this so.

[the remark this answers is not in the document]
Comment (camilamacedo86) [reply pkg/ansible/runner/runner.go]: done 👍

**Comment by camilamacedo86** [comment]
HI @fabianvf, 

Yes, I checked it manually. I reproduce the issue and check it with the POC to ensure that solved.

**Pull request merged**